samples/net/openthread/coap_client/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
I think NCS requires one additional include here:
include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake)

However no idea what for.

What about the NRF_SUPPORTED_BOARDS? Shall we limit the boards as we did not test it to run on platforms other than nrf52xx @LuDuda @rlubos ?

Reply:
> What about the NRF_SUPPORTED_BOARDS? Shall we limit the boards as we did not test it to run on platforms other than nrf52xx

IMO that sounds reasonable. A clear message that board is not supported is always nicer than unspecified build error.

Reply:
ok, how to do that?

Reply:
I've seen something like this in the nfc example samples/nfc/tag_reader/CMakeLists.txt:
# Set supported board
set(NRF_SUPPORTED_BOARDS
  nrf52_pca10040
  nrf52840_pca10056
  )

include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake)
include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)

this was added additionally to specifying platform_whitelist in the yaml file

Reply:
done